docs/vgclog_examples.md

```diff
@@ -0,0 +1,595 @@
+      <mem type="allocate" free="0" total="241565696" percent="0" />
+      <mem type="survivor" free="0" total="26869760" percent="0" />
+    </mem>
+    <mem type="tenure" free="414960320" total="805306368" percent="51">
+      <mem type="soa" free="374694592" total="765040640" percent="48" />
+      <mem type="loa" free="40265728" total="40265728" percent="100" />
+    </mem>
+    <remembered-set count="21474" />
+  </mem-info>
+</gc-start>
+```
+
+The heap memory allocation at the start of the increment is as follows:
+
+- The *allocate* space of the *nursery* area is full, or close to full. The allocation failure was triggered by the lack of available memory in this space.
+- The *survivor* space of the *nursery* area is reported as 'full' to reflect that no available memory is available to allocate to the mutator threads. The entire *survivor* space is reserved for GC operations during the GC increment. 
+- The *tenure* area has 395.7MB (414,960,320B) of free memory available.
+
+The next element `<allocation-stats>` shows a snapshot, which was taken before the cycle started, of the allocation status of the mutator threads. In this example, the thread that used the most memory was `LargeThreadPool-thread-79`.
+
+```xml
+<allocation-stats totalBytes="235362176" >
+  <allocated-bytes non-tlh="32880" tlh="235329296" />
+  <largest-consumer threadName="LargeThreadPool-thread-79" threadId="00000000013F0C00" bytes="6288544" />
+</allocation-stats>
+```
+
+The *scavenge* GC operation is recorded by the `<gc-op>` element; child elements record details about the operation. For example, 
+
+```xml
+<gc-op id="12397" type="scavenge" timems="11.649" contextid="12394" timestamp="2020-10-18T13:35:45.012">
+  <scavenger-info tenureage="7" tenuremask="4080" tiltratio="89" />
+  <memory-copied type="nursery" objects="154910" bytes="6027440" bytesdiscarded="394832" />
+  <memory-copied type="tenure" objects="16171" bytes="562848" bytesdiscarded="3064" />
+  <ownableSynchronizers candidates="10838" cleared="10824" />
+  <references type="soft" candidates="24" cleared="0" enqueued="0" dynamicThreshold="16" maxThreshold="32" />
+  <references type="weak" candidates="390" cleared="269" enqueued="269" />
+  <references type="phantom" candidates="1" cleared="0" enqueued="0" />
+  <object-monitors candidates="132" cleared="0"  />
+</gc-op>
+```
+
+The `<memory-copied>` element indicates that 5.75MB (6,027,440B) of reachable objects were moved by the *scavenge* operation from the allocate space to the survivor space in the nursery area, and 0.54MB(562,848B) were moved to the tenure area. 
+
+The `<scavenger-info>` element shows that the *tenure age* is set to `7`. Any object in the *allocate* space with age below or equal to a tenure age of `7` is copied to the survivor space during this `scavenge`operation. Any object that has been copied between the *allocate* and *survivor* areas more than `7` times is moved to the tenure area. 
+
```

Comment:
you can add what tilt ratio is: allocate / total nursery. Although about aging and tilting in Scavenger we already talk in https://www.eclipse.org/openj9/docs/gc/#gc-operations

Reply:
Adding a link to the gc.md topic, gencon policy section.